inst/qml/AnovaRepeatedMeasures.qml

```diff
@@ -24,6 +24,12 @@ import "./common/classical" as Classical
 
 Form
 {
+	info: qsTr("The repeated Measures ANOVA allows the user to analyze the differences between means, when observations are dependent i.e Within subject factors. It also allows to combine with between subject factors.") + "\n" +
```

Comment:
"It also allows to combine with between subject factors." -> "It also allows combining with between subject factors."